master/plan

```diff
@@ -0,0 +1,87 @@
+KINDS:
```

Comment:
question: should we include this?

Reply:
Nope, it's fine in the feature branch, but should not make it to main